### PR TITLE
Wduraes2

### PR DESCRIPTION
**Pull request by wduraes**
Minor changes to the main README file and adding NXP1060 readme file.

### CODE REVIEW

File: demos/projects/NXP/mimxrt1060/README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Connect an NXP MIMXRT1060-EVK Evaluation kit using Azure IoT Middleware for FreeRTOS
+
+## What you need
+
+* NXP Evaluation Kit: [MIMXRT1060-EVK](https://www.nxp.com/design/development-boards/i-mx-evaluation-and-development-boards/mimxrt1060-evk-i-mx-rt1060-evaluation-kit:MIMXRT1060-EVK)
+
+* USB 2.0 A male to Micro USB male cable
+
+* Wired Ethernet access
```
Comment (hihigupt) [inline demos/projects/NXP/mimxrt1060/README.md:9]: Wired Ethernet access with DHCP

File: demos/projects/NXP/mimxrt1060/README.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+```
+
+To initialize the repo, run the following command:
+
+```bash
+git submodule update --init --recursive --depth 1
+```
+
+## Prepare the device
+To connect the STM DevKit to Azure, you'll modify a configuration file for Wi-Fi and Azure IoT settings, rebuild the image, and flash the image to the device.
```
Comment (hihigupt) [inline demos/projects/NXP/mimxrt1060/README.md:56]: NXP and no Wifi ..:D

File: demos/projects/NXP/mimxrt1060/README.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+1. Connect the Micro USB cable to the Micro USB port on the NXP EVK, and then connect it to your computer. After the device powers up, a solid green LED shows the power status.
+
+1. Use the Ethernet cable to connect the NXP EVK to an Ethernet port.
+
+2. In File Explorer, find the binary file that you created in the previous section and copy it.
+
+3. In File Explorer, find the NXP EVK device connected to your computer. The device appears as a drive on your system with the drive label RT1060-EVK.
+
+4. Paste the binary file into the root folder of the NXP EVK. Flashing starts automatically and completes in a few seconds.
+
```
Comment (hihigupt) [inline demos/projects/NXP/mimxrt1060/README.md:98]: 5. Reset the device once flashing completes.

File: demos/projects/NXP/mimxrt1060/README.md (hunk 1 above)
Comment (wduraes) [reply demos/projects/NXP/mimxrt1060/README.md:9]: good catch. yes.

File: demos/projects/NXP/mimxrt1060/README.md (hunk 2 above)
Comment (wduraes) [reply demos/projects/NXP/mimxrt1060/README.md:56]: that one slipped in my review. thanks!

File: demos/projects/NXP/mimxrt1060/README.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+```
+
+To initialize the repo, run the following command:
+
+```bash
+git submodule update --init --recursive --depth 1
+```
+
+## Prepare the device
+To connect the STM DevKit to Azure, you'll modify a configuration file for Azure IoT settings, rebuild the image, and flash the image to the device.
```
Comment (hihigupt) [inline demos/projects/NXP/mimxrt1060/README.md:56]: copy paste error

Comment (wduraes) [reply demos/projects/NXP/mimxrt1060/README.md]: searched the whole file for any other instance: there's no more.

**Review by hihigupt — APPROVED** [review]

**Pull request merged**